templates/openstack/what-is-openstack.html

```diff
@@ -50,810 +50,343 @@
+    <p>Today OpenStack serves as a de facto standard for building clouds of any type. Native integration with <a href="/kubernetes">Kubernetes</a> and world’s leading chipsets, such as <a href="https://canonical.com/blog/nvidia-vgpu-release">NVIDIA GPUs</a>, make it a perfect candidate for running both traditional and modern, cloud-native applications.</p>
+    '
+    }
+    }
+    ]) }}
 
   {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=false) -%}
     {%- if slot == 'title' -%}
-      <h2>Who uses OpenStack?</h2>
+      <h2>Why OpenStack?</h2>
     {%- endif -%}
 
     {%- if slot == 'description' -%}
       <p>
-        There are
-        <a href="https://www.openstack.org/use-cases/#:~:text=The%20largest%20mobile%20telecom's%20network,OpenStack%20including%20workloads%20like%205G.&text=China%20Mobile's%20telecom%20network%20has,50%2C000%20servers%20in%20the%20network.">thousands of OpenStack users</a>
-        across a broad range of industries. Particularly favored for its
-        stability, security and interoperability. Apart from telecommunications service providers, financial institutions, enterprises, governments, militaries,
-        universities and research institutions all over the world, further
-        adoption of OpenStack has been also seen in healthcare, retail and
-        industrial.
+        OpenStack has been trusted globally by thousands of organizations across all market sectors. Beyond its open source nature, it has been widely adopted around the world thanks to three core pillars in recent years.
       </p>
     {%- endif -%}
 
     {%- if slot == 'list_item_title_1' -%}
-      {{ image(url="https://assets.ubuntu.com/v1/1155d299-kadokawa-logo.png",
-            alt="Kadokawa",
-            width="852",
-            height="82",
-            hi_def=True,
-            loading="lazy") |safe
-      }}
+      <h3 class="p-heading--5">Stable and mature</h3>
     {%- endif -%}
 
     {%- if slot == 'list_item_description_1' -%}
       <p>
-        In 2023, KADOKAWA Connected &mdash; a subsidiary of leading Japanese media conglomerate KADOKAWA Corporation, delivering digital transformation, infrastructure development, and ICT services to the wider KADOKAWA Group &mdash; engaged Canonical to build a new, open-source private cloud as a replacement for their legacy proprietary virtualization platform.
+        OpenStack has been on the market for more than one and half decades now – this means a lot in the software space. Today, OpenStack is more like Linux: it’s a mature project, demonstrating a high level of stability and security as typically required by the most demanding users and industries.
```

Comment:
```suggestion
        OpenStack has been on the market for more than one and a half decades now – this means a lot in the software space. Today, OpenStack is more like Linux: it’s a mature project, demonstrating a high level of stability and security as typically required by the most demanding users and industries.
```

[Flagged this](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit?disco=AAABxqVE83k) with document authors - "one and half" seems unnatural to me